package.json

```diff
@@ -2,25 +2,17 @@
   "private": true,
   "repository": "https://github.com/thunderstore-io/thunderstore-ui",
   "workspaces": [
-    "nextjs",
-    "thunderstore-components"
+    "packages/*"
   ],
-  "dependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
-  },
   "devDependencies": {
     "@microsoft/eslint-formatter-sarif": "^2.1.5",
-    "@types/node": "^14.14.37",
```

Comment:
This package isn't in any of the `package.json`s anymore

Reply:
all of the typechecks work just fine without it at least

Reply:
I think we can just add this if it's needed in the future for `getServerSideProps`